internal/db/queryBuilder.go

```diff
@@ -136,25 +148,38 @@ func (qb *queryBuilder) queryWithRemovals() string {
 	%s
 	)
 
-	SELECT path, mode, size, bytes, packed, deleted
+	SELECT path, mode, size, bytes, packed, deleted, hash
```

Comment:
The hash needs to be returned so that it can be copied into the `target` project